src/Initialization/init_DifferentialEquations.jl

```diff
@@ -31,7 +31,7 @@ function _solve_ensemble(ivp::InitialValueProblem, args...;
     if isnothing(initial_states)
         # sample initial states
         X0 = initial_state(ivp)
-        trajectories = get(kwargs, :trajectories, 100)
+        trajectories = get(kwargs, :trajectories, 10)
```

Comment:
maybe make it a constant `const DEFAULT_TRAJECTORIES = 100` while we are at this?